File: JavaScriptBasics/AdvancedConditionalStatements/Exercise/SummerOutfit.js
function SummerOutfit(input){

      let degrees = Number(input[0]);
      let time  = input[1];
      let outfit;
      let shoes;


      switch(time){
        case "Morning" : 
        if(degrees>=10 && degrees<=18){ outfit="Sweatshirt"; shoes = "Sneakers";}
        if(degrees>=18 && degrees<=24){ outfit="Shirt"; shoes = "Moccasins";}
        if(degrees>=25){ outfit="T-Shirt"; shoes = "Sandals";}break;
        case "Afternoon" : 
        if(degrees>=10 && degrees<=18){ outfit="Shirt"; shoes = "Moccasins";}
        if(degrees>=18 && degrees<=24){ outfit="T-Shirt"; shoes = "Sandals";}
        if(degrees>=25){ outfit="Swim Suit"; shoes = "Barefoot";}break;
        case "Evening" : 
        if(degrees>=10 && degrees<=18){ outfit="Shirt"; shoes = "Moccasins";}
        if(degrees>=18 && degrees<=24){ outfit="Shirt"; shoes = "Moccasins";}
        if(degrees>=25){ outfit="Shirt"; shoes = "Moccasins";}break;
      }

      console.log("It's "+degrees+" degrees, get your "+outfit+" and "+shoes+".")

}

File: JavaBasics/AdvancedConditionalStatements/Exercise/SummerOutfit.java
package EntryModule.AdvancedConditionalStatements.Exercise;

import java.util.Scanner;
public class SummerOutfit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int time = Integer.parseInt(scanner.nextLine());
        String period = scanner.nextLine();

        switch(period){
            case "Morning" : if(10<=time && time<=18){
                System.out.printf("It's %d degrees, get your Sweatshirt and Sneakers.", time);
            }else if(18<time && time<=24){
                System.out.printf("It's %d degrees, get your Shirt and Moccasins.", time);
            }else if(time>=25){
                System.out.printf("It's %d degrees, get your T-Shirt and Sandals.", time);
            }break;
            case "Afternoon" : if(10<=time && time<=18){
                System.out.printf("It's %d degrees, get your Shirt and Moccasins.", time);
            }else if(18<time && time<=24){
                System.out.printf("It's %d degrees, get your T-Shirt and Sandals.", time);
            }else if(time>=25){
                System.out.printf("It's %d degrees, get your Swim Suit and Barefoot.", time);
            }break;
            case "Evening" : if(10<=time && time<=18){
                System.out.printf("It's %d degrees, get your Shirt and Moccasins.", time);
            }else if(18<time && time<=24){
                System.out.printf("It's %d degrees, get your Shirt and Moccasins.", time);
            }else if(time>=25){
                System.out.printf("It's %d degrees, get your Shirt and Moccasins.", time);
            }break;
        }
    }
}
